### PR TITLE
maxBottomBarTabs = 1 for small screens

### PR DESCRIPTION
**Pull request by SamChou19815**
### Summary 

According to our discussion in Slack, we decided to make maxBottomBarTabs be 1 when the screen is small.

### Test Plan 

Screen Shot 2021-03-12 at 21 14 28

### CODE REVIEW

File: src/containers/Dashboard.vue [hunk 1]
```diff
@@ -151,7 +151,11 @@ export default Vue.extend({
     resizeEventHandler() {
       this.isMobile = window.innerWidth <= 440;
       this.isTablet = window.innerWidth <= 878;
-      this.maxBottomBarTabs = window.innerWidth <= 1347 ? 2 : 4;
+      if (window.innerWidth <= 1347) {
+        this.maxBottomBarTabs = window.innerWidth <= 480 ? 1 : 2;
```
Comment (tcho6319) [inline src/containers/Dashboard.vue:155]: Can we define these magic numbers as constants somewhere? I know that most of these values refer to the breakpoint variables:
https://github.com/cornell-dti/course-plan/blob/875d2d17ce41b1e24604128017adfd9a7bf814b4/src/assets/scss/_variables.scss#L37-L40

**Review by tcho6319 — CHANGES_REQUESTED** [review]
Thanks Sam! I left a suggestion for some of the magic numbers we're using. Also, the CI check seems to be failing during the TypeCheck. Once those points are addressed, this PR should be g2g!

File: src/containers/Dashboard.vue [hunk 2]
```diff
@@ -76,6 +76,31 @@ import surfing from '@/assets/images/surfing.svg';
...
+  veryLargeBreakpoint.substring(0, veryLargeBreakpoint.length - 2),
+  10
+);
+
+const getMaxButtonBarTabs = () => {
+  if (window.innerWidth <= smallBreakpointPixels) {
+    return window.innerWidth <= veryLargeBreakpointPixels ? 1 : 2;
+  }
+  return 4;
+};
```
Comment (tcho6319) [inline src/containers/Dashboard.vue:103]: Wouldn't we want to swap for `veryLargeBreakpointPixels` and `smallBreakpointPixels` here?

The behavior is a little wonky:

https://user-images.githubusercontent.com/54298311/111017380-2f266f80-8381-11eb-9b22-4b1f4860e81e.mov

**Review by tcho6319 — CHANGES_REQUESTED** [review]
Almost there! Found a bug from the latest changes.

File: src/containers/Dashboard.vue (hunk 2 above)
Comment (SamChou19815) [reply src/containers/Dashboard.vue]: @tcho6319 fixed now

**Review by tcho6319 — APPROVED** [review]
LGTM 😎

**Pull request merged**